recipes-kernel/linux/linux-raspberrypi.inc

```diff
@@ -103,9 +103,6 @@ do_configure_prepend() {
         kernel_configure_variable OABI_COMPAT y
     fi
 
-    # Set cmdline
-    kernel_configure_variable CMDLINE "\"${CMDLINE}\""
```

Comment:
will I be able to see it in zcat proc/config.gz | grep CMDLINE ?

Reply:
No, but I think the CONFIG_CMDLINE variable in /proc/config.gz is meaningless anyway. You'd want to look at `/proc/cmdline` for the command line actually used at runtime.

Reply:
I'm planning to leave my change as-is. That ok?